src/redux/reducers/surveyReducer.js

```diff
@@ -3,9 +3,16 @@ import { combineReducers } from 'redux';
 const surveyScore = (state = [], action) => {
   switch (action.type) {
     case 'SET_SCORE':
-      return [
-        ...state, action.payload
-      ];
+      return [ ...state, action.payload ];
+    case 'REMOVE_BEHAVIOR':
+      return state.filter(survey => {
```

Comment:
[This](https://redux.js.org/recipes/structuringreducers/refactoringreducersexample#extracting-case-reducers) is a better way to keep your reducer clean and all of the functions in reducers out of the reducer's main function. They usually get pretty big.

-- Nikko

Reply:
Thanks Nikko, I will try it :)